services/frontend-service/src/ui/components/ReleaseVersion/ReleaseVersion.scss

```diff
@@ -0,0 +1,33 @@
+.release-version {
```

Comment:
please use the same formatting as in other scss files: 4 spaces to indent.

Reply:
Done.